config/envs/rhpam-authoring-ha.yaml

```diff
@@ -382,7 +419,7 @@ others:
                           "value": "8888"
                         }
                       ],
-                      "image": "amq-broker-72-openshift:1.3",
+                      "image": "registry.redhat.io/amq-broker-7/amq-broker-73-openshift:7.3",
```

Comment:
Should we put these in the constants.go file? Thoughts, @tchughesiv ?

Reply:
I think better add it do constants file.
Should I add this under TemplateConstants struct?

Reply:
Yes, I think so, same pattern as for our own product images, I would say... and to address Tommy's other point, maybe also similarly look an image stream first as we do, and only if not found, create a local image stream pointing to source image? Thoughts, @tchughesiv ?

Reply:
the existing operator logic should handle it already

Reply:
but, yes ... needs to be changed in yaml

Reply:
I forget to update this one.